src/content/en/updates/2018/08/chrome-69-media-updates.md

```diff
@@ -0,0 +1,321 @@
+`chrome://flags/#enable-experimental-web-platform-features`.
+
+[Intent to Experiment](https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/ITzZ_yx4bF8) &#124;
+[Chromestatus Tracker](https://www.chromestatus.com/feature/5652917147140096) &#124;
+[Chromium Bug](https://crbug.com/709348)
+
+## MSE PTS/DTS compliance {: #pts }
+
+Buffered ranges and duration values are now reported by Presentation Time Stamp
+(PTS) intervals, rather than by Decode Time Stamp (DTS) intervals in [Media
+Source Extensions (MSE)].
+
+When MSE was new, Chrome’s implementation was tested against WebM and MP3, some
+media stream formats where there was no distinction between PTS and DTS. And
+it was working fine until ISO BMFF (aka MP4) was added. This container
+frequently contains out-of-order presentation versus decode time streams (for
+codecs like H.264 for example) causing DTS and PTS to differ. That caused
+Chrome to report (usually just slightly) different buffered ranges and duration
+values than expected. This new behavior will roll out gradually in Chrome 69
+and make its MSE implementation compliant with the [MSE specification].
+
+<figure>
+  <img src="/web/updates/images/2018/08/pts-dts.png"
+       alt="PTS/DTS">
+  <figcaption>
+    <b>Figure 2.</b>
+    PTS/DTS
+  </figcaption>
+</figure>
+
+This change affects `MediaSource.duration` (and consequently
+`HTMLMediaElement.duration`), `SourceBuffer.buffered` (and consequently
+`HTMLMediaElement.buffered)`, and `SourceBuffer.remove(start, end)`.
+
+If you’re not sure which method is used to report buffered ranges and duration
+values, you can go to the internal `chrome://media-internals` page and search for
+"ChunkDemuxer: buffering by PTS" or  "ChunkDemuxer: buffering by DTS" in the
+logs.
+
+[Intent to Implement](https://groups.google.com/a/chromium.org/d/msg/blink-dev/I6oGJLym-Tk/Z46le9l4CQAJ) &#124;
+[Chromium Bug](https://crbug.com/718641)
+
+## Handling of media view intents on Android Go {: #media_intent_handler }
+
+[Android Go] is a lightweight version of Android designed for entry-level
+smartphones. To that end, it does not necessarily ship with some media-viewing
+applications, so if a user tries to open a downloaded video for instance, they
+won’t have any applications to handle that intent.
+
+To fix this, Chrome 69 on Android Go now listens for media-viewing intents so
+users can view downloaded audio, videos, and images. In other words, it takes
+the place of the missing viewing applications.
+
+<figure>
+  <img src="/web/updates/images/2018/08/media-intent-handler.png"
+       alt="Media intent handler">
+  <figcaption>
+    <b>Figure 3.</b>
+    Media intent handler
+  </figcaption>
+</figure>
+
+Note that this Chrome feature is enabled on all Android devices running Android
+O and onwards with 1 GB of RAM or less.
+
+[Chromium Bug](https://crbug.com/718641)
+
+## Removal of “stalled” events for media elements using MSE {: #stalled }
+
```

Comment:
Francois,

Can you do me a favor can you add this macro right after the heading (line 272):

{% include "web/updates/_shared/see-all-dep-rem.html" %}

You can see what it looks like [here](https://developers.google.com/web/updates/2018/06/chrome-68-deps-rems).